packages/server/test/e2e/5_headless_spec.ts

```diff
@@ -4,6 +4,30 @@ import Fixtures from '../support/helpers/fixtures'
 describe('e2e headless', function () {
   e2e.setup()
 
+  describe('ELECTRON_RUN_AS_NODE', () => {
+    beforeEach(function () {
+      process.env.ELECTRON_RUN_AS_NODE = 'true'
+    })
+
+    // cypress run --headless
+    e2e.it('tests in headless mode pass', {
```

Comment:
this test does not seem to assert that ELECTRON_RUN_AS_NODE does not spawn an Electron process

Reply:
to add custom test code, you can pass an `onRun` property here that receives an `exec` function as an argument to start Cypress. see other e2e tests in this directory for examples.

Reply:
Thanks for the tip about onRun. The strategy I took to test that electron was not spawned was to enable debug logs for `cypress:xvfb,cypress:server:run` and then checking stdout for debug messages that I expect to be printed when electron is not run. Do that feel sufficient to you? I couldn't think of any better testing strategies

Reply:
Yes!